### DISCUSSION TITLE
Cannot add repository into community exchange

### DISCUSSION THREAD

**Discussion by Nambers** [community/community#103076] [GitHub Education]
### Hi everyone,

Hi there, here is the error message "Unfortunately a service Community Exchange depends on is not available. Please try again at a later time.".

**Comment by gil-son** [comment]
If that's persist

You must to contact [Github Support](https://support.github.com/) 

I hope I was helpfull. Good luck!

**Comment by Aisuko** [comment]
I hit same issue. It hasn't worked several days.

**Reply by saeedahmadicp** [reply]
It's the same issue with me as well. I tried several times but the same errors popped up to me as well.

**Reply by pritish384** [reply]
Same here

**Reply by saeedahmadicp** [reply]
It's working now for me, @pritish384. I believe the issue will be resolved for you as well. I suggest checking it every few days. I'm not sure of the exact cause of the issue, but it should be fixed soon.

**Reply by pritish384** [reply]
I have contact the Github support regarding this and they have transferred this case to engineering team

**Reply by saeedahmadicp** [reply]
@pritish384 ok, that's best

**Comment by samus-aran** [comment]
Hi everyone! 👋 

Wanted to ask if anyone is still experiencing issues with adding their repository to Community Exchange?

**Reply by pritish384** [reply]
Yes

**Reply by samus-aran** [reply]
Thanks @pritish384 - I believe you are communicating with the support team on this within your raised support ticket. We'll continue communications on your issue from there.

**Reply by hominsu** [reply]
still here

**Reply by ahujaesh** [reply]
Its not working for me either.

**Reply by PossiblePanda** [reply]
Not working for me still :/

**Reply by BananaJeanss** [reply]
Having the same problem.

**Reply by PossiblePanda** [reply]
@samus-aran Any clue when this will be fixed?

**Comment by SUBHADIPMAITI-DEV** [comment]
Is anyone else currently facing this problem? 'Unfortunately, a service Community Exchange depends on is not available. Please try again at a later time.'

**Reply by BananaJeanss** [reply]
I think the problem only happens when you try to use "Learn and collaborate", try only using "learn" or "collaborate" instead
